### DISCUSSION TITLE
Move popup search screen to top along with taskbar?

### DISCUSSION THREAD

**Discussion by nanrector** [valinet/ExplorerPatcher#3887] [Q&A]
Hello! 
I've used Explorer Patcher to move my taskbar to the top of my huge tv/monitor. 
However, when I click to search, that big popup box (not sure what that's called) still pops up from the bottom left. 
Is there a way to also move it to the top?

Thanks!
Nancy 

![searchresult](https://github.com/user-attachments/assets/f5c2f6da-675e-4cdb-b336-0c1818732192)

**Comment by pyrates999** [comment]
duplicate of: https://github.com/valinet/ExplorerPatcher/issues/1803

Please close this.
